api/modules/feedback/views.py

```diff
@@ -30,3 +31,44 @@ def add_feedback(request):
 
     success_message = "Sucessfully added new feedback."
     return Response(success_message, status=status.HTTP_201_CREATED)
+
+
+@api_view(['GET'])
+def get_feedback(request, feedback_id):
+    """
+    Returns the feedback pertaining to a certain feedback id
+    :param request:
+    :return: 400 if incorrect parameters are sent or database request failed
+    :return: 201 successful
+    """
+    try:
+        user_feedback = Feedback.objects.get(pk=feedback_id)
+
```

Comment:
Add a check here to verify if the current user is associated with the feedback or not
This ensures someone doesn't open up any other person's feedback